packages/client/manifest.json

```diff
@@ -4322,6 +4322,103 @@
       }
     ]
   },
+  "attachmentsinglefield": {
+    "name": "Single Attachment",
+    "icon": "Attach",
+    "styles": ["size"],
+    "requiredAncestors": ["form"],
+    "editable": true,
+    "size": {
+      "width": 400,
+      "height": 200
+    },
+    "settings": [
+      {
+        "type": "field/attachment_single",
```

Comment:
We need to define what settings component should be rendered for this new type `field/attaachment_single`. The existing `FormFieldSelect` will handle it, so we just need to add 
```js
  "field/attachment_single": FormFieldSelect,
```
to `componentSettings.js`.
Without this, the `field` setting is never rendered.